src/main/scala/org/opennetworkinsight/FlowPostLDA.scala

```diff
@@ -86,12 +89,24 @@ object FlowPostLDA {
         val words = sc.broadcast(l_words)
 
         logger.info("loading data")
-        val rawdata = sc.textFile(config.inputPath)
-
-        val datanoheader = FlowWordCreation.removeHeader(rawdata)
-        val datagood = datanoheader.filter(line => line.split(",").length == 27)
-
-        val data_with_time = datagood.map(_.trim.split(",")).map(FlowWordCreation.addTime)
+        val rawdata: RDD[String] = {
+          val flowDataFrame = sqlContext.parquetFile(config.inputPath)
+            .filter("trhour BETWEEN 0 AND 23 AND  " +
+              "trminute BETWEEN 0 AND 59 AND  " +
+              "trsec BETWEEN 0 AND 59")
+            .select("trhour",
+              "trminute",
+              "trsec",
+              "sip",
+              "dip",
+              "sport",
+              "dport",
+              "ipkt",
+              "ibyt")
+          flowDataFrame.map(_.mkString(","))
+        }
+
+        val data_with_time = rawdata.map(_.trim.split(",")).map(FlowWordCreation.addTime)
```

Comment:
okay, I see what you did here (making minimal changes consistent with existing code) but it does seem kind of weird to pack everything into  string and then have to split it and parse it later... 

we can think about a cleaner approach in a future change...